CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.2.5]
```

Comment:
will it be used immediately? if no, then not bumping version should be fine

Reply:
Yes it will be use immediately, there is a ticket to implements this fix.